dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java

```diff
@@ -220,6 +220,8 @@ public void run(DockstoreWebserviceConfiguration configuration, Environment envi
 
         final ObjectMapper mapper = environment.getObjectMapper();
         mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        // For when we want to globally ignore all json properties with null value during serialization
```

Comment:
Could you open an issue explaining this and link to it from here?

Reply:
https://github.com/ga4gh/dockstore/issues/1092